scripts/pkgx-env.sh

```
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# shellcheck shell=bash
+
+# Source this file from a shell or IDE environment-file setting.
+if [[ -n "${BASH_VERSION:-}" ]]; then
+  script_file="${BASH_SOURCE[0]}"
+elif [[ -n "${ZSH_VERSION:-}" ]]; then
+  script_file="${(%):-%x}"
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with (. Double check syntax.

(SC2296)